src/lib/plugin.js

```diff
@@ -208,7 +209,11 @@ class FiveBellsLedger extends EventEmitter2 {
     }
 
     const reconnect = reconnectCore(() => {
-      return new WebSocket(wsUri)
+      return new WebSocket(wsUri, null, {
+        headers: {
+          authorization: 'Bearer ' + options.authToken
```

Comment:
I thought the reason we were using the token in the URI was because the browser doesn't support these websocket headers. Won't this break browser support?